files/dashboards/cdsdashboards/templates/editdashboard.html

```diff
@@ -0,0 +1,305 @@
+
+                    <p><label for="conda_env">Conda Env:</label>
+                        {% if allow_custom_conda_env %}
+                            <input type="text" name="conda_env" value="{{ conda_env | escape }}" 
+                                {% if all_conda_envs %} list="conda_env_list" {% endif %}
+                            />
+                        {% endif %}
+                        {% if all_conda_envs %}
+                            <{% if allow_custom_conda_env %}datalist{% else %}select name="conda_env"{% endif %} id="conda_env_list">
+                                <option value="" {% if conda_env == '' %}selected{% endif %}>None / Default</option>
+                                {% for ce in all_conda_envs %}
+                                    <option value="{{ce}}" {% if conda_env == ce and not allow_custom_conda_env %}selected{% endif %}>{{ce}}</option>
+                                {% endfor %}
+                            </{% if allow_custom_conda_env %}datalist{% else %}select{% endif %}>
+                        {% endif %}
+                    </p>
+
+                {% endif %}
+
+                {% if errors.conda_env %}
+    
+                <p class="error">{{ errors.conda_env | escape }}</p>
+    
+                {% endif %}
+
+                <p><label for="start_path">Relative Path to a file or folder: </label></p>
+                
+    
+                <p><input type="text" name="start_path" value="{{ dashboard_start_path | escape }}" id="dashboard-start-path" /></p>
+                <p> <i>Relative to your Jupyter tree or Git root</i></p>
+                
+                <!-- JupyterHub Spawner Options -->
+                <h3>Server Options</h3>
+                <div>
+                    <label for="memory" class="col-2 col-form-label">JupyterLab Memory (GB)</label>
+                    <div class="col-10">
+                        <input type="number" value="1" id="memory" name="memory">
+                    </div>
+                </div>
+                <div>
+                    <label for="nprocs" class="col-2 col-form-label">JupyterLab CPUs</label>
+                    <div class="col-10">
+                        <input type="number" value="1" id="nprocs" name="nprocs">
+                    </div>
+                </div>
+                <div>
+                    <label for="partition" class="col-2 col-form-label">Slurm Partition</label>
+                    <div class="col-10">
+                        <select class="form-select" aria-label="Slurm Queue" id="partition" name="partition">
+                            <option value="general">general</option>
+                            {% for item in groups %}
+                            {% if item.startswith('partition-')%}
+                                <option value="{{ item[10:] }}">{{ item[10:] }}</option>
+                            {% endif %}
+                            {% endfor %}
```

Comment:
@costrouc I'm not sure why this doesn't work currently.  The example partition doesn't show up currently.  Do you have any ideas why that might be?